source/core/TDCodecVia.cpp

```diff
@@ -2206,10 +2296,11 @@ VIREO_FUNCTION_SIGNATUREV(FlattenToJSON, FlattenToJSONParamBlock)
     } else {
         formatter.FormatData(arg[0]._paramType, arg[0]._pData);
     }
-    if (formatter.HasError()) {
+    Int32 errCode = formatter.GetError();
+    if (errCode) {
```

Comment:
I don't think we are properly adding call chain information or the append string as in other calls.